.gitsplit.yml

```
@@ -66,6 +66,8 @@ splits:
     target: "https://${GH_TOKEN}@github.com/opentelemetry-php/contrib-instrumentation-installer.git"
   - prefix: "src/Logs/Monolog"
     target: "https://${GH_TOKEN}@github.com/opentelemetry-php/contrib-logger-monolog.git"
+  - prefix: "src/Propagation/Instana"
+    target: "https://${GH_TOKEN}@github.com/opentelemetry-php/opentelemetry-propagation-instana.git""
```

For consistency with the other existing propagator contrib repositories, maybe:
```suggestion
    target: "https://${GH_TOKEN}@github.com/opentelemetry-php/contrib-propagator-instana.git""
```

---

addressed.

---

It looks the same to me?